### PR TITLE
chore(components/indicators): split expansion indicator styles

### PR DESCRIPTION
**Pull request by Blackbaud-PaulCrowder**
[AB#2767140](https://dev.azure.com/blackbaud/f565481a-7bc9-4083-95d5-4f953da6d499/_workitems/edit/2767140)

### CODE REVIEW

**Comment by blackbaud-sky-build-user** [comment]
[Storybook preview](https://blackbaud.github.io/skyux-pr-preview/2453/storybook/)

Component Storybooks:

- [action-bars-storybook](https://blackbaud.github.io/skyux-pr-preview/2453/storybooks/action-bars-storybook/)
- [ag-grid-storybook](https://blackbaud.github.io/skyux-pr-preview/2453/storybooks/ag-grid-storybook/)
- [angular-tree-component-storybook](https://blackbaud.github.io/skyux-pr-preview/2453/storybooks/angular-tree-component-storybook/)
- [avatar-storybook](https://blackbaud.github.io/skyux-pr-preview/2453/storybooks/avatar-storybook/)
- [colorpicker-storybook](https://blackbaud.github.io/skyux-pr-preview/2453/storybooks/colorpicker-storybook/)
- [data-manager-storybook](https://blackbaud.github.io/skyux-pr-preview/2453/storybooks/data-manager-storybook/)
- [datetime-storybook](https://blackbaud.github.io/skyux-pr-preview/2453/storybooks/datetime-storybook/)
- [errors-storybook](https://blackbaud.github.io/skyux-pr-preview/2453/storybooks/errors-storybook/)
- [flyout-storybook](https://blackbaud.github.io/skyux-pr-preview/2453/storybooks/flyout-storybook/)
- [forms-storybook](https://blackbaud.github.io/skyux-pr-preview/2453/storybooks/forms-storybook/)
- [indicators-storybook](https://blackbaud.github.io/skyux-pr-preview/2453/storybooks/indicators-storybook/)
- [inline-form-storybook](https://blackbaud.github.io/skyux-pr-preview/2453/storybooks/inline-form-storybook/)
- [layout-storybook](https://blackbaud.github.io/skyux-pr-preview/2453/storybooks/layout-storybook/)
- [lists-storybook](https://blackbaud.github.io/skyux-pr-preview/2453/storybooks/lists-storybook/)
- [lookup-storybook](https://blackbaud.github.io/skyux-pr-preview/2453/storybooks/lookup-storybook/)
- [modals-storybook](https://blackbaud.github.io/skyux-pr-preview/2453/storybooks/modals-storybook/)
- [pages-storybook](https://blackbaud.github.io/skyux-pr-preview/2453/storybooks/pages-storybook/)
- [phone-field-storybook](https://blackbaud.github.io/skyux-pr-preview/2453/storybooks/phone-field-storybook/)
- [progress-indicator-storybook](https://blackbaud.github.io/skyux-pr-preview/2453/storybooks/progress-indicator-storybook/)
- [split-view-storybook](https://blackbaud.github.io/skyux-pr-preview/2453/storybooks/split-view-storybook/)
- [tabs-storybook](https://blackbaud.github.io/skyux-pr-preview/2453/storybooks/tabs-storybook/)
- [text-editor-storybook](https://blackbaud.github.io/skyux-pr-preview/2453/storybooks/text-editor-storybook/)
- [theme-storybook](https://blackbaud.github.io/skyux-pr-preview/2453/storybooks/theme-storybook/)
- [tiles-storybook](https://blackbaud.github.io/skyux-pr-preview/2453/storybooks/tiles-storybook/)
- [toast-storybook](https://blackbaud.github.io/skyux-pr-preview/2453/storybooks/toast-storybook/)

Apps:

- [code-examples](https://blackbaud.github.io/skyux-pr-preview/2453/code-examples/)
- [integration](https://blackbaud.github.io/skyux-pr-preview/2453/integration/)
- [playground](https://blackbaud.github.io/skyux-pr-preview/2453/playground/)

File: libs/components/indicators/src/lib/modules/expansion-indicator/expansion-indicator.component.ts [hunk 1]
```diff
@@ -6,7 +6,10 @@ import { Component, Input } from '@angular/core';
  */
 @Component({
   selector: 'sky-expansion-indicator',
-  styleUrls: ['./expansion-indicator.component.scss'],
```
Comment (Blackbaud-SteveBrush) [inline libs/components/indicators/src/lib/modules/expansion-indicator/expansion-indicator.component.ts:9]: I'm new to this; are we keeping the old stylesheet (why not delete)?

Comment (Blackbaud-PaulCrowder) [reply libs/components/indicators/src/lib/modules/expansion-indicator/expansion-indicator.component.ts:9]: It looks like I missed deleting the original stylesheet after resolving some merge conflicts. It's gone now.

Comment (Blackbaud-SteveBrush) [reply libs/components/indicators/src/lib/modules/expansion-indicator/expansion-indicator.component.ts:9]: Cool.

**Review by Blackbaud-SteveBrush — APPROVED** [review]

**Comment by blackbaud-sky-build-user** [comment]
Cherry-pick successful! :cherries: :tada: See https://github.com/blackbaud/skyux/pull/2454

**Pull request merged**